docs/docs-content/enterprise-version/upgrade/upgrade-k8s/airgap.md

```diff
@@ -0,0 +1,432 @@
+
+- Access to the x86 Linux jumpbox or bastion host with connectivity to the target platform where you have installed
+  Palette.
+
+- An OCI registry such as [Harbor](https://goharbor.io/) or [AWS ECR](https://aws.amazon.com/ecr/) configured and
+  available to store the new Palette images and packages.
+
+- Access to the latest Palette airgap setup binary. Refer to [Access Palette](/enterprise-version/#access-palette) for
+  more details.
+
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) and [`helm`](https://helm.sh/docs/intro/install/)
+  available in your system.
+
+- Access to the `kubeconfig` file of the target Kubernetes cluster. You must be able to interact with the cluster
+  through `kubectl` and have sufficient permissions to upgrade Palette. We recommend using a role with the cluster-admin
+  permissions.
+
+- `unzip` or a similar tool available in your system.
+
+- Access to the latest Palette Helm Chart. Refer to [Access Palette](/enterprise-version/#access-palette) for more
+  details.
+
+## Upgrade
+
+1.  Log in to the Linux environment from which you can access your self-hosted airgap Palette instance.
+2.  Authenticate with your OCI registry and get credentials to your `spectro-packs` and `spectro-images` repositories.
+
+    :::info
+
+    Be mindful of the auth token timeout. Uploading Palette images and packages to the OCI registry can take about an
+    hour. If your auth token expires before everything is uploaded, you will need to re-authenticate and start the
+    upload again.
+
+    :::
+
+    Select one of the following tabs for instructions on how to authenticate with Harbor and AWS ECR.
+
+    <Tabs>
+
+    <TabItem value="harbor" label="Harbor" default>
+
+    Use the following command template to authenticate with your Harbor OCI registry through `oras`. Refer to the
+    [oras login](https://oras.land/docs/commands/oras_login/) docs for more details about the available CLI flags and
+    examples.
+
+    ```shell
+    oras login X.X.X.X --user 'yourUserNameHere' --password 'yourPasswordHere'
+    ```
+
+    :::tip
+
+    If your Harbor registry has a self-signed certificate, use the `--insecure` flag.
+
+    ```shell
+    oras login X.X.X.X --insecure --user 'yourUserNameHere' --password 'yourPasswordHere'
+    ```
+
+    :::
+
+    </TabItem>
+
+    <TabItem value="aws-ecr" label="AWS ECR">
+
+    To get the authentication commands for your specific registries, navigate to the AWS ECR console and click **View
+    push commands**. Refer to the
+    [AWS ECR Authentication](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-cli.html#cli-authenticate-registry)
+    docs for more details.
+
+    Alternatively, you can use the following command templates:
+
+    - Authenticate with your private `spectro-packs` registry.
+
+      ```shell
+      aws ecr get-login-password --region xxxxx | oras login --username AWS --password-stdin 1234567890.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
It might be good to replace the `1234567890.dkr.ecr.us-east-1.amazonaws.com` with a placeholder and explicitly tell the user to replace the value. Otherwise, they will probably just copy and paste.